lib/ts/recipe/passwordless/api/implementation.ts

```diff
@@ -2,797 +2,146 @@ import { APIInterface } from "../";
 import { logDebugMessage } from "../../../logger";
 import AccountLinking from "../../accountlinking/recipe";
 import Session from "../../session";
```

Comment:
please reset this to what was in the master branch.. (as discussed before)

Reply:
Done